day1/arraysum.c

```diff
@@ -0,0 +1,19 @@
+#include<stdio.h>
+
+int main()
+{      
+        int i,sum=0;
+        int arr[10];
+        printf("\n enter the numbers in array :");
+        for(i=0;i<10;i++)
```

Comment:
always initialize a new variable in for loop like `for(int i=0;i<10;i++)`

Reply:
What is the complexity of your program? there is no need to write two for loops instead you could have done it in one loop itself
    ```
  for(i=0;i<10;i++)
        {
			int input;
			scanf("%d",&input);
			sum+=input;
		}
```